libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationUIBinders.kt

```diff
@@ -5,12 +5,44 @@ import com.mapbox.navigation.dropin.binder.infopanel.InfoPanelTripProgressBinder
 import com.mapbox.navigation.dropin.component.maneuver.ManeuverViewBinder
 import com.mapbox.navigation.dropin.component.roadlabel.RoadNameViewBinder
 import com.mapbox.navigation.dropin.component.speedlimit.SpeedLimitViewBinder
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
 
-class NavigationUIBinders(
-    val speedLimit: UIBinder = SpeedLimitViewBinder(),
-    val maneuver: UIBinder = ManeuverViewBinder(),
-    val roadName: UIBinder = RoadNameViewBinder(),
-    val infoPanelTripProgressBinder: UIBinder = InfoPanelTripProgressBinder(),
-    val infoPanelHeaderBinder: UIBinder? = null,
-    val infoPanelContentBinder: UIBinder? = null
-)
+internal class NavigationUIBinders {
+
+    private val _speedLimit: MutableStateFlow<UIBinder> = MutableStateFlow(SpeedLimitViewBinder())
+    private val _maneuver: MutableStateFlow<UIBinder> = MutableStateFlow(ManeuverViewBinder())
+    private val _roadName: MutableStateFlow<UIBinder> = MutableStateFlow(RoadNameViewBinder())
+    private val _infoPanelTripProgressBinder: MutableStateFlow<UIBinder> =
+        MutableStateFlow(InfoPanelTripProgressBinder())
+    private val _infoPanelHeaderBinder: MutableStateFlow<UIBinder?> = MutableStateFlow(null)
+    private val _infoPanelContentBinder: MutableStateFlow<UIBinder?> = MutableStateFlow(null)
+
+    val speedLimit: StateFlow<UIBinder> get() = _speedLimit.asStateFlow()
+    val maneuver: StateFlow<UIBinder> get() = _maneuver.asStateFlow()
+    val roadName: StateFlow<UIBinder> get() = _roadName.asStateFlow()
+    val infoPanelTripProgressBinder: StateFlow<UIBinder> get() =
+        _infoPanelTripProgressBinder.asStateFlow()
+    val infoPanelHeaderBinder: StateFlow<UIBinder?> get() = _infoPanelHeaderBinder.asStateFlow()
+    val infoPanelContentBinder: StateFlow<UIBinder?> get() = _infoPanelContentBinder.asStateFlow()
+
+    fun applyCustomization(c: Customization) {
```

Comment:
```suggestion
    fun applyCustomization(customization: Customization) {
```

Reply:
This is nitpicking!. This method is fine as it is. Renaming this argument to `customization` doesn't increase readability.

Reply:
This is nitpicking!. This method is fine as it is. Renaming this argument to `customization` doesn't increase readability.